src/style.css

```diff
@@ -1,3 +1,8 @@
-.hello {
-  color: red;
+* {
+  margin: 0;
+  padding: 0;
+}
+
+footer {
+  padding: 2rem 2rem;
 }
```

Comment:
You can try to place the footer at the bottom of the page 

footer {
  bottom: 0;
  position: fixed;
  width: 100%;
  z-index: 2;
}

Reply:
Thanks for the tip. I've never used this approach before for footers.